src/Aspire.Hosting.Azure/AzureResourceExtensions.cs

```diff
@@ -273,24 +273,6 @@ public static IResourceBuilder<AzureCosmosDBEmulatorResource> UseGatewayPort(thi
         });
     }
 
-    /// <summary>
-    /// Adds an Azure Redis resource to the application model.
-    /// </summary>
-    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
-    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
-    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<AzureRedisResource> AddAzureRedis(this IDistributedApplicationBuilder builder, string name)
```

Comment:
We're losing the ability to make this work for development. I suggest this PR (since it's the first one add an AzureResourceSurrogate annotation and the AzureProvisioner be updated to consume it. That way we can still provision in dev.

This is the first instance of `RunAsAzureRedis`.

Reply:
Yeah I think that is probably what we'll need to do to preserve this behavior.